pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -268,6 +268,11 @@ func parseBaseOS(getconfigCtx *getconfigContext,
 	baseOS := config.GetBaseos()
 	if baseOS == nil {
```

Comment:
Just to understand it right - do we really have a situation with baseOsConfig == null? For some reason, I thought this field should also contain something. Or is it used only during the updates?

Reply:
The controller can send an empty list. Happens with the commercial controller if you do zcli edge-node eveimage-remove.
It was confusing to me that it didn't get fully removed when I used that recently. So just removing a source of confusion here.